sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java

```diff
@@ -181,9 +181,15 @@ public void setFullscreenButtonShowing(boolean showing) {
 
   @ReactMethod
   public void onClosedCaptionUpdateRequested(ReadableMap parameters) {
-    final String languageName = parameters.hasKey("language") ? parameters.getString("language") : null;
-    double curTime = _player.getPlayheadTime() / 1000d;
-    WritableMap params = BridgeMessageBuilder.buildClosedCaptionUpdateParams(_player, languageName, curTime);
+    WritableMap params;
+    // ignore the request if cc is not available
+    if (_player == null || _player.getCurrentItem() == null || !_player.getCurrentItem().hasClosedCaptions()) {
+       params = Arguments.createMap();
```

Comment:
this is fine, but for the sake of discussion - What do you think about not even sending the callback response?  any reason you chose to send empty params instead?  what would an empty map do on the react side?